worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/FactoryConverter.java

```diff
@@ -42,29 +46,48 @@
 import org.enginehub.piston.inject.Key;
 
 import java.util.List;
+import java.util.function.Consumer;
 import java.util.function.Function;
 
 public class FactoryConverter<T> implements ArgumentConverter<T> {
 
+    @AutoAnnotation
+    private static ClipboardMask clipboardMask() {
+        return new AutoAnnotation_FactoryConverter_clipboardMask();
+    }
+
     public static void register(WorldEdit worldEdit, CommandManager commandManager) {
         commandManager.registerConverter(Key.of(Pattern.class),
-            new FactoryConverter<>(worldEdit, WorldEdit::getPatternFactory, "pattern"));
+            new FactoryConverter<>(worldEdit, WorldEdit::getPatternFactory, "pattern", c -> {}));
         commandManager.registerConverter(Key.of(Mask.class),
-            new FactoryConverter<>(worldEdit, WorldEdit::getMaskFactory, "mask"));
+            new FactoryConverter<>(worldEdit, WorldEdit::getMaskFactory, "mask", c -> {}));
         commandManager.registerConverter(Key.of(BaseItem.class),
-            new FactoryConverter<>(worldEdit, WorldEdit::getItemFactory, "item"));
+            new FactoryConverter<>(worldEdit, WorldEdit::getItemFactory, "item", c -> {}));
+
+        commandManager.registerConverter(Key.of(Mask.class, clipboardMask()),
+                new FactoryConverter<>(worldEdit, WorldEdit::getMaskFactory, "mask",
+                        context -> {
+                            try {
+                                context.setExtent(context.getSession().getClipboard().getClipboard());
```

Comment:
note to self: get holder transform and wrap clipboard region in transform extent